packages/sequelize-transparent-cache/lib/index.js

```diff
@@ -9,21 +12,24 @@ module.exports = function (client) {
         return classMethods(client, this)
       }
 
+      modelClass.manualCache = function (customId) {
+        return manualCacheMethods(client, this, customId)
+      }
+
       modelClass.prototype.cache = function () {
         return instanceMethods(client, this)
       }
 
       return modelClass
     },
     classMethods: {
-      cache () {
-        return classMethods(client, this)
-      }
+      cache () { return classMethods(client, this) }
+    },
+    manualCacheMethods: {
```

Comment:
it's here to support older sequelize versions, `manualCacheMethods` is not a valid sequelize settings option, so `manualCache` should be part of `classMethods`